app/src/features/mappings/FhirResourceStep.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from "react";
+
+import { Icon } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+import {
+  Container,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  makeStyles,
+  Typography,
+  TextField,
+} from "@material-ui/core";
+import CheckIcon from "@material-ui/icons/Check";
+import SearchIcon from "@material-ui/icons/Search";
+import clsx from "clsx";
+import { useTranslation } from "react-i18next";
+
+import { Resource } from "services/api/generated/api.generated";
+
+//Mock
+const FhirResources = [
+  "Account",
+  "ActivityDefinition",
+  "AdverseEvent",
+  "AllergyIntolerance",
+  "Appointment",
+];
+
+const useStyles = makeStyles((theme) => ({
+  searchBarContainer: {
+    paddingInline: theme.spacing(25),
+    paddingBlock: theme.spacing(2),
+  },
+  listItem: {
+    border: `1px solid ${theme.palette.divider}`,
+    borderRadius: 5,
+    boxShadow: `0 1px 5px ${theme.palette.divider}`,
+    marginBottom: theme.spacing(2),
+    display: "flex",
+    alignItems: "center",
+  },
+  icon: {
+    color: theme.palette.text.secondary,
+    marginRight: theme.spacing(1),
+  },
+  flameIcon: {
+    fill: "#CC7831",
+  },
+}));
+
+type FhirResourceStepProps = {
+  mapping: Partial<Resource>;
+  onChange?: (mapping: Partial<Resource>) => void;
+};
+
+const FhirResourceStep = ({
```

Comment:
ok same issue as my comment above I guess. I will try to fix this asap (just created arkhn/deployment#194)